src/components/Common/Fields/ComboBoxField/ComboBoxField.tsx

```diff
@@ -0,0 +1,39 @@
+import { useField, type UseFieldProps } from '@/components/Common/Fields/hooks/useField'
+import type { ComboBoxItem } from '@/components/Common/UI/ComboBox/ComboBox'
+import { ComboBox } from '@/components/Common/UI/ComboBox/ComboBox'
+
+interface ComboBoxFieldProps
+  extends Omit<React.ComponentProps<typeof ComboBox>, 'name' | 'onChange' | 'onBlur' | 'options'>,
```

Comment:
Then not omit options

```suggestion
  extends Omit<React.ComponentProps<typeof ComboBox>, 'name' | 'onChange' | 'onBlur'>,
```